FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Keep_Alive/core_mqtt_config.h

```diff
@@ -75,6 +74,41 @@ extern void vLoggingPrintf( const char * pcFormatString,
  * macro sets the limit on how many simultaneous PUBLISH states an MQTT
  * context maintains.
  */
-#define MQTT_STATE_ARRAY_MAX_COUNT    10U
+#define MQTT_STATE_ARRAY_MAX_COUNT      10U
+
+/**
+ * @brief The maximum duration between non-empty network reads while
+ * receiving an MQTT packet via the #MQTT_ProcessLoop or #MQTT_ReceiveLoop
+ * API functions.
+ *
+ * When an incoming MQTT packet is detected, the transport receive function
+ * may be called multiple times until all of the expected number of bytes of the
+ * packet are received. This timeout represents the maximum polling duration that
+ * is allowed without any data reception from the network for the incoming packet.
+ *
+ * @note For this demo, the timeout value is configured to zero as the demo uses a
+ * dummy timer function (of #MQTTGetCurrentTimeFunc_t) that always returns zero.
+ * It is REQUIRED to set the the timeout to zero when using a dummy timer function
+ * that always zero.
+ */
+#define MQTT_RECV_POLLING_TIMEOUT_MS    0U
+
+/**
+ * @brief The maximum duration between non-empty network transmissions while
+ * sending an MQTT packet via the #MQTT_ProcessLoop or #MQTT_ReceiveLoop
+ * API functions.
+ *
+ * When sending an MQTT packet, the transport send function may be called multiple
+ * times until all of the required number of bytes are sent.
+ * This timeout represents the maximum duration that is allowed for no data
```

Comment:
maximum time to wait for what?

Reply:
Maximum time to wait for a single byte to be transmitted over the network over multiple calls to the transport send function.

Reply:
Can you make the code comment read more like this explanation?